Feat: Adds stop method to SDKs

g-charan opened the pull request:
Implements the `stop` method in both JavaScript and Python SDKs, enabling users to signal the completion of a process or node in FlowMetr workflows.

This method constructs a URL with provided parameters (node ID, run ID, logs) and sends a GET request to the FlowMetr backend.

closes #4

g-charan committed:
Feat: Adds stop method to SDKs

Implements the `stop` method in both JavaScript and Python SDKs, enabling users to signal the completion of a process or node in FlowMetr workflows.

This method constructs a URL with provided parameters (node ID, run ID, logs) and sends a GET request to the FlowMetr backend.

hansende reviewed (changes requested):
Suggested Improvements:
- use the url variable instead of hardcoded url
- requests has a params variable which can be used like: 
    params = {
    "run_id": run_id,
       ...
    }
    res = requests.get(f"https://flowmetr.com/hooks/{self.flow_id}", params=params)
- local vars
  you don't need to e.g. use self.node_id in your method. just use node_id. it is a local var
- include error handling with try..catch
-  use a timeout on requests

g-charan committed:
Refactors FlowMetr SDK for improved reliability

Replaces the previous implementation of the FlowMetr SDK in both Javascript and Python with a more robust and reliable version.
The updated SDK includes:
- A private helper method for sending events consistently.
- Request timeouts to prevent indefinite hanging.
- Improved error handling and logging.
- Standardized parameter passing using URLSearchParams (JS) and requests.get params (Python).

This refactoring enhances the SDK's stability and provides better feedback in case of errors or network issues.
